presto-accumulo/src/test/java/io/prestosql/plugin/accumulo/TestingAccumuloServer.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.accumulo;
+
+import org.apache.accumulo.core.client.AccumuloException;
+import org.apache.accumulo.core.client.AccumuloSecurityException;
+import org.apache.accumulo.core.client.Connector;
+import org.apache.accumulo.core.client.ZooKeeperInstance;
+import org.apache.accumulo.core.client.security.tokens.PasswordToken;
+import org.testcontainers.containers.FixedHostPortGenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import java.time.Duration;
+
+import static java.lang.String.format;
+
+public class TestingAccumuloServer
+{
+    private static final int ACCUMULO_MASTER_PORT = 9999;
+    private static final int ACCUMULO_TSERVER_PORT = 9997;
+    private static final int ZOOKEEPER_PORT = 2181;
+
+    private static final TestingAccumuloServer instance = new TestingAccumuloServer();
+
+    private final FixedHostPortGenericContainer<?> accumuloContainer;
+
+    public static TestingAccumuloServer getInstance()
+    {
+        return instance;
+    }
+
+    private TestingAccumuloServer()
+    {
+        // TODO Change to a prestodev/accumulo container once released
+        accumuloContainer = new FixedHostPortGenericContainer<>("datacatessen/accumulo:1");
+        accumuloContainer.withFixedExposedPort(ACCUMULO_MASTER_PORT, ACCUMULO_MASTER_PORT);
+        accumuloContainer.withFixedExposedPort(ACCUMULO_TSERVER_PORT, ACCUMULO_TSERVER_PORT);
+        accumuloContainer.withExposedPorts(ZOOKEEPER_PORT);
+        accumuloContainer.waitingFor(Wait.forHealthcheck().withStartupTimeout(Duration.ofMinutes(10)));
+
+        // No need for an explicit stop since this server is a singleton
```

Comment:
Using a singleton for now is fine as a transition from the old way, but we should move to a model where the Accumulo cluster is managed by `AccumuloQueryRunner`, the same way we do for other testing services. Shutting down resources immediately is useful since the CI machines can be resource constrained, and using a non-singleton allows things like testing multiple versions, testing with different config, etc.

Add a TODO here to make this not a singleton.

Reply:
> but we should move to a model where the Accumulo cluster is managed by AccumuloQueryRunner,

Let's create a github issue for this and add TODO comment (with link to the issue to the code).